gatsby-config.js

```diff
@@ -41,6 +42,25 @@ const plugins = [
       path: path.join(__dirname, 'content')
     }
   },
+  {
+    resolve: `gatsby-source-github-api`,
+    options: {
+      // token: required by the GitHub API
+      token: process.env.GITHUB_TOKEN,
```

Comment:
is it build time? what happens if it's not defined? (like local dev machine)

Reply:
It is build time, and currently the site throws when the token is removed. I can fix it such that this doesn't happen and build without the token present just use dummy data, it'll just take a little extra logic to handle that.

Reply:
Fixed! Now stars displays as 8888 when no GitHub API token is given.

Reply:
is it easy enough to skip and return empty value?